### PR TITLE
docs: sanity checks

### PR DESCRIPTION
**Pull request by herrardo**
Sanity Checks

# Pull request template
Fix bugs that Luisa found on this document https://searchbroker.atlassian.net/wiki/spaces/ED/pages/3639738373/Sanity+check+notes for the first 3 sections:

- Integration links
- Callout Syntaxis
- Breaking Space

## Motivation and context
Correct links and update callout viz.

- [ ] Dependencies. If any, specify:
- [X] Open issue. If applicable, link: EX-5035

## Type of change

- [ ] Bug fix (non-breaking change that fixes an issue)
- [ ] New feature (non-breaking change that adds functionality)
- [ ] Breaking change (fix or feature that causes existing functionality to not work as expected)
- [X] Change requires a documentation update

## What is the destination branch of this PR?

- [X] `Main`
- [ ] Other. Specify:

## How has this been tested?

Tests performed according to [testing guidelines](./contributing/tests.md): 

## Checklist:

- [ ] My code follows the **[style guidelines](./CONTRIBUTING.md#style-guides)** of this project.
- [X] I have performed a **self-review** on my own code.
- [ ] I have **commented** my code, particularly in hard-to-understand areas.
- [ ] I have made corresponding changes to the **[documentation](./CONTRIBUTING.md#documentation-style-guide)**.
- [ ] My changes generate **no new warnings**.
- [ ] I have added **tests** that prove my fix is effective or that my feature works.
- [ ] New and existing **unit tests pass locally** with my changes.

### CODE REVIEW

File: packages/x-components/static-docs/build-search-ui/web-x-components-integration-guide.md [hunk 1]
```diff
@@ -13,9 +13,12 @@ tags:
 In this tutorial, you’ll learn the basics to integrate the Interface&nbsp;X&nbsp;Components library
 in your own project to craft enticing Vue search experiences for your shop in a matter of minutes.
 
-::: interact If you are looking to use the ready-to-go project Interface&nbsp;X&nbsp;Archetype as
-your starting point, see
+::: interact
+
+If you are looking to use the ready-to-go project Interface&nbsp;X&nbsp;Archetype as your starting
+point, see
 **[Interface X Archetype Development](/develop-empathy-platform/build-search-ui/web-archetypedevelopment-guide.md)**.
```
Comment (javieri-empathy) [inline packages/x-components/static-docs/build-search-ui/web-x-components-integration-guide.md:20]: Should this link also be changed to 
```suggestion
**[Interface X Archetype Development](web-archetype-development-guide.md)**.
```

**Review by javieri-empathy — CHANGES_REQUESTED** [review]

File: packages/x-components/static-docs/build-search-ui/README.md [hunk 2]
```diff
@@ -22,13 +22,12 @@ Start building your search and discovery experience in a few simple steps:
...
-  and how Interface&nbsp;X works.
-- **[Integrate the Interface X Components](/develop-empathy-platform/build-search-ui/README.md#integrate-x-components)**
-  in your store web application.
-- **[Use & configure the Interface X Components](/develop-empathy-platform/build-search-ui/README.md#use-configure-the-x-components)**
-  in your project.
+- **Discover more about the [Interface X ecosystem](#the-interface-x-ecosystem)** and how
+  Interface&nbsp;X works.
+- **[Integrate the Interface X Components](#integrate-interface-x-components)** in your store web
+  application.
+- **[Use & configure the Interface X Components](#use-configure-the-interface-x-components)** in
```
[the remark this answers is not in the document]
Comment (herrardo) [reply packages/x-components/static-docs/build-search-ui/README.md]: Suggestion applied

File: packages/x-components/static-docs/build-search-ui/web-x-components-integration-guide.md (hunk 1 above)
Comment (herrardo) [reply packages/x-components/static-docs/build-search-ui/web-x-components-integration-guide.md]: Suggestion applied

File: packages/x-components/static-docs/build-search-ui/README.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (javieri-empathy) [reply packages/x-components/static-docs/build-search-ui/README.md]: I meant that this link won't work both in GitHub and EmpathyDocs at the same time. GitHub renders the id of the header with an extra `-` that [EmpathyDocs](https://docs.empathy.co/develop-empathy-platform/build-search-ui/?utm_term=x#use-configure-the-interface-x-components) does not. I used the suggestion to highlight the difference, but I didn't intend you to commit it 😅, so roll it back if you can please.

```suggestion
- **[Use & configure the Interface X Components](#use-configure-the-interface-x-components)** in
```

I just wanted to raise awareness between this differences between GitHub and EmpathyDocs markdown renderers.

File: packages/x-components/static-docs/build-search-ui/README.md [hunk 3]
```diff
@@ -22,13 +22,12 @@ Start building your search and discovery experience in a few simple steps:
...
-  and how Interface&nbsp;X works.
-- **[Integrate the Interface X Components](/develop-empathy-platform/build-search-ui/README.md#integrate-x-components)**
-  in your store web application.
-- **[Use & configure the Interface X Components](/develop-empathy-platform/build-search-ui/README.md#use-configure-the-x-components)**
-  in your project.
+- **Discover more about the [Interface X ecosystem](#the-interface-x-ecosystem)** and how
+  Interface&nbsp;X works.
+- **[Integrate the Interface X Components](#integrate-interface-x-components)** in your store web
+  application.
+- **[Use & configure the Interface X Components](#use--configure-the-interface-x-components)** in
```
Comment (javieri-empathy) [inline packages/x-components/static-docs/build-search-ui/README.md:29]: I meant that this link won't work both in GitHub and EmpathyDocs at the same time. GitHub renders the id of the header with an extra `-` that [EmpathyDocs](https://docs.empathy.co/develop-empathy-platform/build-search-ui/?utm_term=x#use-configure-the-interface-x-components) does not. I used the suggestion to highlight the difference, but I didn't intend you to commit it 😅, so roll it back if you can please.

```suggestion
- **[Use & configure the Interface X Components](#use-configure-the-interface-x-components)** in
```

I just wanted to raise awareness between this differences between GitHub and EmpathyDocs markdown renderers.

**Review by javieri-empathy — APPROVED** [review]

**Review by lydiafp — CHANGES_REQUESTED** [review]

**Review by lydiafp — COMMENTED** [review]
Please see comments. There are few things that need to be changed

File: packages/x-components/static-docs/build-search-ui/README.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (herrardo) [reply packages/x-components/static-docs/build-search-ui/README.md:29]: Fixed

File: packages/x-components/static-docs/build-search-ui/web-x-components-integration-guide.md [hunk 4]
```diff
@@ -72,9 +77,12 @@ npm install --save @empathyco/x-components @empathyco/x-types @empathyco/x-adapt
...
-adapter. :::
+::: warning
+
+If you are using the Empathy search API and the Empathy Search Adapter (`x-adapter`), make sure you
+have imported the `reflect-metadata` polyfill before executing any code of the search adapter.
+
+:::
 
 Next, construct the search adapter. You will need the search adapter in the
 [xPlugin configuration](#_3-configure-the-xplugin).
```
[the remark this answers is not in the document]
Comment (herrardo) [reply packages/x-components/static-docs/build-search-ui/web-x-components-integration-guide.md:88]: Fixed

File: packages/x-components/static-docs/build-search-ui/web-x-components-integration-guide.md [hunk 5]
```diff
@@ -130,22 +142,22 @@ Then, **configure** the xPlugin. It has two key options you need to configure:
    If you are not using the Empathy Search API, you need to build your own adapter. See
    [Configure the search adapter](#_2-configure-the-search-adapter).
```
[the remark this answers is not in the document]
Comment (herrardo) [reply packages/x-components/static-docs/build-search-ui/web-x-components-integration-guide.md:143]: Fixed

**Review by lydiafp — APPROVED** [review]
@herrardo  - Changes approved!

**Review by javieri-empathy — APPROVED** [review]

**Pull request merged**